### ISSUE TITLE
Error when fetch's init argument isn't set

### ISSUE THREAD

**Issue by mchill** [nutboltu/storybook-addon-mock#39]
The second argument of `fetch()` is optional: https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch#syntax

This addon's mocked `Request` object tries to access the `options.method` property without checking that `options` is set, and so throws an error if `options` is undefined.
```javascript
export function Request(input, options) {
    if (typeof input === 'object') {
        this.method = options.method || input.method || 'GET'; // fails if `options` isn't set
        this.url = input.url;
    } else {
        this.method = options.method || 'GET'; // also fails
        this.url = input;
    }
}
```

**Comment by schemata-issimo** [comment]
I'm having this same issue, causing react-refresh to to get stuck because of missing options parameter.

The pull request fixes it though, thanks!

**Comment by nutboltu** [comment]
@mchill thanks for raising this issue and fix it in a short time. I am making few other changes and there will be a new version bump. I'll update you all soon.

**Comment by nutboltu** [comment]
@mchill This issue is fixed in version `1.1.0`. Closing this issue.
